maquetado/scss/styles.scss

```diff
@@ -1,4 +1,25 @@
-.title {
-  color: red;
+h1,
+p {
+  display: block;
+}
+
+h1 {
+  font-size: 2em;
+  font-weight: bold;
+}
+
+.container {
```

Comment:
use a more descriptive name like "flex"

Reply:
but flex is the value accepted by `display` property :thinking: